skimage/color/colorlabel.py

```diff
@@ -195,34 +196,110 @@ def _label2rgb_overlay(label, image=None, colors=None, alpha=0.3,
     return result
 
 
-def _label2rgb_avg(label_field, image, bg_label=0, bg_color=(0, 0, 0)):
+def _apply_per_channel(func, image):
+    """
+    Apply a function to each channel of an image.
+
+    Parameters
+    ----------
+    func : callable
+        A function that takes an array to another array of the same shape.
+    image : array
+        A single or multichannel image.
+
+    Returns
+    -------
+    out : array, same shape and type as `image`
+    """
+    if len(image.shape) not in [2, 3]:
+        raise ValueError('Image must be 2d or 3d array')
+    if len(image.shape) == 2:
+        return func(image)
+    else:
+        out = np.zeros(image.shape)
+        n = image.shape[2]
+        for k in range(n):
+            out[..., k] = func(image[..., k])
+        return out
+
+
+def _get_means_from_contiguous_regions(label_field, image):
+    """
+    Aggregates pixel values with respect to
```

Comment:
```suggestion
    """Aggregate pixel values with respect to
```
Reference: https://peps.python.org/pep-0257/#multi-line-docstrings